.doecode.yml

```
@@ -0,0 +1,81 @@
+---
+code_id: 18101
+site_ownership_code: "BNL"
+open_source: true
+repository_link: "https://github.com/bluesky/bluesky.git"
+project_type: "OS"
+software_type: "S"
+official_use_only: {}
+developers:
+- email: "scampbell@bnl.gov"
+  orcid: "0000-0001-7079-0878"
+  first_name: "Stuart"
+  last_name: "Campbell"
+  middle_name: ""
+  affiliations:
+  - "Brookhaven National Lab. (BNL), Upton, NY (United States). National Synchrotron\
+    \ Light Source II (NSLS-II)"
+- email: "dallan@bnl.gov"
+  orcid: ""
+  first_name: "Daniel"
+  last_name: "Allan"
+  middle_name: ""
+  affiliations:
+  - "Brookhaven National Lab. (BNL), Upton, NY (United States). National Synchrotron\
+    \ Light Source II (NSLS-II)"
+- email: "tcaswell@bnl.gov"
+  orcid: ""
+  first_name: "Thomas"
+  last_name: "Caswell"
+  middle_name: ""
+  affiliations:
+  - "Brookhaven National Lab. (BNL), Upton, NY (United States). National Synchrotron\
+    \ Light Source II (NSLS-II)"
+- email: "mrakitin@bnl.gov"
+  orcid: ""
```

Per https://orcid.org/my-orcid?orcid=0000-0003-3685-852X and https://github.com/mrakitin/
```suggestion
  orcid: "0000-0003-3685-852X"
```